noxfile.py

```diff
@@ -80,7 +80,7 @@ def test(session: nox.Session):
     )
     covfile = Path(session.create_tmp(), ".coverage")
     more_args = []
-    if session.python == "3.11":
+    if session.python in ("3.11", "3.12"):
```

Comment:
Sets are more optimized for `__contains__()` than tuples are, but meh. 

```suggestion
    if session.python in {"3.11", "3.12"}:
```

Reply:
For small element counts tuples will likely beat sets.

The set notation, on the other hand, is cleaner and less easy to screw up (when changing `("3.11", "3.12")` to `("3.12")` for example).